homework-submission/src/checkbox.js

```diff
@@ -0,0 +1,11 @@
+import React,{Component} from 'react';
+
+class CheckBoxes extends Component {
```

Comment:
- This class deals with a single checkbox. Therefore its name should be `Checkbox`. Note that checkbox is a one word (check your dictionary), therefore the `b` of box should be lowercase. The filename should be `Checkbox.js`.
- You are not setting the `value` of the checkbox. Therefore, even though a todo is marked as 'done', your checkbox doesn't show a check mark.